kotlin-react/src/main/kotlin/react/RBuilder.kt

```diff
@@ -120,3 +120,29 @@ fun <P : RProps> forwardRef(handler: RBuilder.(RProps, RRef) -> Unit): RClass<P>
         buildElements { handler(props, ref) }
     }
 }
+
+typealias FunctionalComponent<P> = (props: P) -> dynamic
+
+/**
+ * Get functional component from [func]
+ */
+fun <P : RProps> functionalComponent(
```

Comment:
Does it replace `rFunction`?
https://github.com/JetBrains/kotlin-wrappers/blob/master/kotlin-react/src/main/kotlin/react/ReactComponent.kt#L78-L85